katalog/thanos/README.md

```diff
@@ -0,0 +1,95 @@
+# Thanos
+
+Basically Thanos is an opensource Prometheus setup that allows to have 2 important features:
+
+1. high availability on Prometheus (setting more prometheus replicas)
+2. long term storage capacity relying on an external object storage
+
+This is the simplified diagram of a Thanos architecture:
+
+![thanos-architecture](images/thanos-arch.png)
+
+
+# Setup in Fury Module
+
+We designed 2 different possible setup accordingly with the 2 features listed above:
+
+```bash
+thanos #this is the setup with only high-availability feature
+thanos-with-store #this include the storing feature with an object storage
+thanos-components #this directory contains all the thanos components and is intended to not be used directly
+```
+So, you'll need to add to your `bases` the `thanos` or `thanos-with-store` depending on your needs.
+Remember than `thanos-with-store` is a superset that include also the high availability feature. 
+
```

Comment:
Manual patching to prometheus CRD is needed to add the sidecar

Reply:
right @nutellinoit !!! i need to ad it to caveat section! Thanks!

Reply:
done!